superset/assets/webpack.config.js

```diff
@@ -102,8 +109,12 @@ const config = {
     },
   },
   resolve: {
-    extensions: ['.js', '.jsx'],
+    alias: {
+      '@': path.resolve(APP_DIR, './src'),
```

Comment:
wanted to gauge thoughts on making this more readable like `@src` or `@root`, etc. instead of _just_ `@`.

I don't care that strongly, but do think more readability is generally good.

Reply:
Okey-dokey. Renaming it to `@src` shortly. :)

Reply:
sgtm, don't have a strong pref / like them equally, too 😃